kong-ngx-build

```diff
@@ -387,6 +402,81 @@ main() {
   succ "Build finished in $SECONDS seconds. Enjoy!"
 }
 
+parse_version() {
+  [[ -z $1 ]] && fatal 'missing arg $1 when invoking parse_version()'
+  [[ -z $2 ]] && fatal 'missing arg $2 when invoking parse_version()'
+
+  # remove non-version parts of a version, ie foobar-1.2.3a.5-rc1 -> 1.2.3a.5
```

Comment:
We should update this comment saying it only removes the prefix on versions! Which I think is ok, if we are not going to support dash releases

Reply:
I just rewrote the parsing utility, and as a bonus, got rid of the Perl dependency, made the parsing  70% faster, and consider `-rcN` suffixes!